src/ui/components/UserReview/index.js

```diff
@@ -27,36 +28,40 @@ type Props = {|
 type InternalProps = {|
   ...Props,
   i18n: I18nType,
+  id: String,
```

Comment:
This isn't an InternalProp. In fact it isn't a prop at all. You can remove this.